canard.h

```diff
@@ -42,8 +42,8 @@ extern "C" {
 #endif
 
 /// Libcanard version. API will be backwards compatible within the same major version.
-#define CANARD_VERSION_MAJOR                        0
-#define CANARD_VERSION_MINOR                        2
+#define CANARD_VERSION_MAJOR                        1
+#define CANARD_VERSION_MINOR                        0
```

Comment:
Are we sure we're ready to make this commitment? Just asking.

Reply:
As I see it, until the uavcan-v1.0 is merged back to master it is still considered as a development branch and API may still be changed. I thought it was natural to update it here due to all the other renamings, but if you don't want to update the major version until we are 100% finished and ready for merge back to master that is fine for me.

Reply:
Okay, it makes sense, let's keep it at 1.0.

Reply:
While we're changing version numbers, may we also implement explicit differentiation between the version numbers of the library itself and of the specification that it implements? Like this: https://github.com/thirtytwobits/libuavcan/pull/1/files

Libcanard version 1.0 implements UAVCAN version 1.0, which is essentially a coincidence; at some point there may be libcanard v2.0 implementing UAVCAN v1.0, causing confusion among users unless we version it more explicitly. I don't think it's necessary to also specify the minor version of the standard (this decision can be reviewed later).